deepchecks/checks/performance/performance_report.py

```diff
@@ -64,12 +67,12 @@ def add_condition_score_not_less_than(self, min_score: float):
         Args:
             min_score (float): Minimal score to pass.
         """
-        name = f'Metrics score is not less than {min_score}'
+        name = f'Score is not less than {min_score}'
 
         def condition(result, min_score):
             not_passed = {k: v for k, v in result.items() if v < min_score}
             if not_passed:
-                details = f'Metrics with lower score: {not_passed}'
+                details = f'Scores that are less than {min_score}: {not_passed}'
```

Comment:
no need to repeat the `min_score` since it's already in the name

Reply:
updated